rules/windows/builtin/application/mssqlserver/win_mssql_dangerous_query.yml

```
@@ -0,0 +1,27 @@
+title: MSSQL Dangerous Query
+id: 18dae75f-b0c8-4076-95bc-9a1a56e7001e
+status: experimental
+description: Detects the invocation of an "ALTER TABLE" or "DROP TABLE" transaction in order to delete stored data.
+references:
+    - https://learn.microsoft.com/en-us/sql/t-sql/statements/drop-table-transact-sql?view=sql-server-ver16
+author: Daniel Degasperi '@d4ns4n_'
+date: 2025-03-03
+tags:
+    - attack.exfiltration
+    - attack.impact
+    - attack.t1485
+logsource:
+    product: windows
+    service: application
+    definition: 'Requirements: MSSQL audit policy must be enabled in order to receive this event (event id 33205)'
+detection:
+    selection:
+        Provider_Name|contains: 'MSSQL'
```

```suggestion
        Provider_Name: 'MSSQLSERVER$AUDIT'
```
can't we match this specific provider name? Is there a reason for the `|contains`?